data_structures/src/chain.rs

```diff
@@ -53,6 +53,9 @@ pub struct ChainInfo {
 
     /// Checkpoint of the last block in the blockchain
     pub highest_block_checkpoint: CheckpointBeacon,
+
+    /// Checkpoint of the last vrf otuput in the blockchain
+    pub highest_vrf_output: CheckpointBeacon,
```

Comment:
This field would benefit from having a different type, so we never accidentally use a `CheckpointBeacon` (epoch + block hash) instead of (epoch + vrf hash). We would need to change it in `VrfMessage::block_mining` and `VrfMessage::data_request`, and all the methods that use them. Not sure how to call this new type though.